modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -40,8 +40,8 @@ Default configuration
 | Name | Type | Description | Default Value | Required
 | Name | String | Name for this configuration. Connectors reference the configuration with this name. | None | Yes
 | Api Definition | String |  API definition. | None | Yes
-| Kafka Configs | Array of <<KafkaConfig>>. | None | No |  
-| Anypoint MQ Configs | Array of <<AnypointMqConfig>>. | None | No |  
+| Kafka Configs | Array of <<KafkaConfig>>. | Configuration settings for Kafka integration. | None | No  
+| Anypoint MQ Configs | Array of <<AnypointMqConfig>>. | Configuration settings for Anypoint MQ integration. | None | No
 | Name | String | Identifier of this element used to reference it in other components. | None | Yes
```

Comment:
For october we need to add two more protocols - solace, salesforce pubsub. I will share the reference doc in the slack channel.

Reply:
OK! Thanks!
Adding these comments to the following PR: https://github.com/mulesoft/docs-apikit/pull/201/files